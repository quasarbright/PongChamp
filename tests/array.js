let x = [2, 3, [4, 5]];
println(x[0]);
println(x[2]);

x[1] = 4;
println(x[1]);

println(x.length);
x.append("a");
println(x);
println(x.length);