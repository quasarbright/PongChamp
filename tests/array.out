CNumber 2
[CNumber 4,CNumber 5]
CNumber 4
CNumber 3
[CNumber 2,CNumber 4,CPointer 10,CString "a"]
CNumber 4
